com.github.jerempw.gnonograms.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "com.github.jeremypw.gnonograms",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "com.github.jeremypw.gnonograms",
+
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--filesystem=home",
+      "--device=dri"
+    ],
+
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+
+    "modules": [
+        {
+            "name": "gnonograms",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/jeremypw/gnonograms.git",
+                    "branch": "add-flatpak-manifest"
```

Comment:
this is going to need to be pinned to a specific commit from that branch for reproducibility.

Reply:
OK, will do.